srcpkgs/libmypaint/template

```diff
@@ -23,6 +23,9 @@ libmypaint-devel_package() {
 		vmove usr/include
 		vmove usr/lib/pkgconfig
 		vmove "usr/lib/*.so"
+		if test -e "${DESTDIR}/usr/share/gir-1.0"; then
```

Comment:
use `if [ "$build_option_gir" ]; then` instead

Reply:
The changed if condition apparently is true even when the gir-1.0 directory doesn't exist, which makes the vmove fail on several architectures.

Reply:
I removed the check altogether now.